.ci/metrics/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.12
```

Comment:
I prefer using the full URL for the base image so we know exactly where the container is coming from.